scripts/get_git_commit.py

```diff
@@ -14,9 +14,6 @@
         .decode("utf-8")
     )
 except Exception:
-    print("==========================================================================")
-    print("  WARNING: You are not in a Git repository or do not have Git installed.")
-    print("           The version string will be incorrect.")
-    print("==========================================================================")
+    revision = "NOT_GIT"
```

Comment:
Won't this remove any git revision set through an environment variable?

Reply:
Not sure about if it's about customized case where there's an extra line in the `platformio.ini`. At that case since `platformio.ini` is to be modified anyway, could just comment out the line calling this Python script and write the line for `GIT_REV`.

For other cases the script just set `GIT_REV` string to `"NOT_GIT"` when can't get current commit hash.

Reply:
The web firmware tool has the git commit hash, but it doesn't use it as a git repo, so this will always throw an exception. It's best that the web firmware tool doesn't modify any files it doesn't need to as that's more to maintain, but setting an environment variable is really easy.

Reply:
fixed with the latest commit